.helm_staging.yml

```diff
@@ -1,3 +1,7 @@
+service:
+    type: NodePort
+    port: 8080
```

Comment:
Please set this to `80`, as current port value. It will work with `8080` but that will force ingress LB to be recreated.

Reply:
Changed as requested